jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```diff
@@ -94,8 +94,8 @@ ProcessSessionId=Process Session Id
 WorkItemId=Work Item Id
 TaskSuccessfullyForwarded=Task was successfully forwarded
 ReminderSentTo=Reminder was successfully sent to {0}
-PleaseEnterUserOrAGroupToDelegate=Please enter a user or a group to delegate the task
-PleaseEnterUserOrAGroupToForward=Please enter a user or a group to forward the task
+PleaseEnterUserOrAGroupToDelegate=Please enter a user to delegate the task
+PleaseEnterUserOrAGroupToForward=Please enter a user to forward the task
```

Comment:
The forward message is correct, only delegate needs to be changed.

Reply:
I find PleaseEnterUserOrAGroupToForward and PleaseEnterUserOrAGroupToDelegate has never been used